src/features/ListAllMovies/ui/MoviesList.tsx

```diff
@@ -35,21 +46,35 @@ export const MoviesList: FC<MovieListProp> = ({ type, limit }) => {
         return <div>Error loading movies. Please try again later.</div>;
     }
 
+    const items = data?.pages.flatMap((page) => splitToRows(page.items, 6));
```

Comment:
I've noticed that reelcinemas apply `movie details` logic only for desktops view, but for mobile and tables when user clicks on a particular movie it redirects to make an order page. Let's also do like this 
And I have a question how does 6 items in a row look for big/small sizes?